rules/windows/lateral_movement_scheduled_task_target.toml

```diff
@@ -13,7 +13,39 @@ license = "Elastic License v2"
 name = "Remote Scheduled Task Creation"
 note = """## Triage and analysis
 
-Decode the base64 encoded tasks actions registry value to investigate the task configured action."""
+### Investigating Creation of Remote Scheduled Tasks
+
+[Scheduled tasks](https://attack.mitre.org/techniques/T1053/) are a great mechanism used for persistence and executing programs. These features can
```

Comment:
Since the technique is already built into the ATT&CK breakout in the UI, we probably do not need to link it here. I do however, like linking a resource here, so perhaps a reference to Microsoft or other explanatory reference.

Reply:
Yeh, makes sense.  Just changed it to a more general description from Microsoft.